string/E6.jsh

```diff
@@ -0,0 +1,48 @@
+System.out.println("Your method:");
+/list mixString
+Thread.sleep(5000);
+System.out.println();
+System.out.println("And we will test it for this calls:");

```

Comment:
This calls znaczy To woła. Testowanie dla "tych wywołań" będzie brzmiało "these calls".